dataplane/dplanerc/interface.go

```diff
@@ -404,6 +404,17 @@ func (ni *Reconciler) addLAGMember(ctx context.Context, intf ocInterface, member
 		if err := ni.ifaceMgr.LinkSetDown(memberLink); err != nil {
 			log.Warningf("failed to set link %v down: %v", intf.name, err)
 		}
+		for i := 0; i < 3; i++ { // Ensure link is down before continuing.
+			memberLink, err := ni.ifaceMgr.LinkByIndex(memberData.hostifIfIndex)
+			if err != nil {
+				break
```

Comment:
I suppose the error is due to non-existing link. Should we return error or we just don't care?

Reply:
changed to continue